raisinbread/RB_files/RB_test_battery.sql

```diff
@@ -1,38 +1,39 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `test_battery`;
 LOCK TABLES `test_battery` WRITE;
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `CohortID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (131,'aosi',1,2147483647,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
+-- AgeMaxDays maximum value is 99999
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `CohortID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (131,'aosi',1,36500,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
 INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `CohortID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (132,'aosi',1,4299,'Y','Visit',NULL,'V2',NULL,NULL,NULL);
```

Comment:
I do not know AOSI instrument, is there any particular reason for this one to stay 4299?

Reply:
ok, I will fix it to 36500 as well.

Reply:
It's already below the limit, is there any reason to change it? (it's the "Autism Observation Scale for Infants" and 4299 works out to just under 12yo so I don't think there's any reason to raise it.)

Reply:
Ok I did not know what the acronym stood for. Thanks. I think 100y is enough for the rest.